app/src/main/java/org/torproject/android/ui/v3onionservice/clientauth/ClientAuthActivity.java

```diff
@@ -36,6 +36,9 @@
     private ContentResolver mResolver;
     private ClientAuthListAdapter mAdapter;
 
+    static final String CLIENT_AUTH_FILE_EXTENSION = ".auth_private",
+            CLIENT_AUTH_SAF_MIME_TYPE = "*/*";
```

Comment:
highly annoying the newish storage access framework on android only lets you query via MIME types instead of say file extension